textile/features.textile

```diff
@@ -2154,6 +2173,13 @@ class Message: // TM*
   id: String // TM2a
   name: String? // TM2g
   timestamp: Time // TM2f
+  action: enum // TM2j
```

Comment:
the IDL defines enums explicitly, eg look at `PresenceAction`

Reply:
Updated to be consistent with `PresenceAction`